lib/muffin.rb

```diff
@@ -1,3 +1,5 @@
+require "active_support"
+require "active_support/core_ext"
```

Comment:
As we talked about it, having ActiveSupport around makes live ofter way more easy and this thing will be used within rails anyway.